backend/compact-connect/docs/README.md

```diff
@@ -35,12 +35,12 @@ For your convenience, use of this feature is included in the [Postman Collection
 
```

Comment:
General question: I see many pylint suppressions which gives me a little pause. I know we've discussed this in a prior PR, but there seem to be more in this PR than average. Is there a better way?

Reply:
I'm definitely open to alternatives, including alternative linting tools, if there is an alternative that is less prone to false-positives. I am trying to find the balance of not over-commenting in a way that is difficult to maintain, while not leaving legitimate suppressions unjustified.

Reply:
Makes complete sense. I have no solutions at the moment, so we can table for a future time. (Regardless, not necessarily less prone to false-positives, but very fast is [Ruff](https://docs.astral.sh/ruff/), which I've been using recently and have enjoyed.)